.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

No need to escape these values, as no reserved characters are used. You might also want to disable automatic rebasing as this can lead to a lot of CI backlogging. Also, settting a larger PR limit is a good idea as Dependabot might otherwise hide updates.

```suggestion
  - package-ecosystem: npm
    directory: /
    open-pull-requests-limit: 999
    rebase-strategy: disabled
    schedule:
      interval: daily
```

---

thanks, will add this as suggested.